Sources/Sentry/SentrySdkPackage+Equality.h

```diff
@@ -0,0 +1,13 @@
+#import "SentrySdkPackage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentrySdkPackage (Equality)
```

Comment:
`m`: If we use a Swift struct for just the data of SentrySDKPackage you get isEqual for free.